stm32-modules/heater-shaker/scripts/requirements.txt

```diff
@@ -0,0 +1,9 @@
+cycler==0.10.0
```

Comment:
Have you used pandas? I suggest we use it if we get more into plotting and data analysis. It's obscenely powerful.

Reply:
I have not, I should check it out at some point. This stuff isn't really controlled at all and doesn't ship so I just kind of went with what I knew.